jenkins-scripts/dsl/gz-collections.yaml

```diff
@@ -441,67 +441,67 @@ collections:
       - name: gz-cmake
         major_version: 5
         repo:
-          current_branch: gz-cmake4
+          current_branch: gz-cmake5
```

Comment:
We should do the opposite here. Currently, Jetty should be the exact copy of Ionic.